src/Pages/eunjinlee/Main/Comments/Comments.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import './Comments.scss'
+import Comment from './Comment/Comment'
+
+class Comments extends React.Component {
+
+  constructor() {
+    super()
+    this.state = {
+      commentList: [],
+      value: '',
+      openComments: false,
+    }
+  }
+  
+  componentDidMount = () => {
+    fetch('http://localhost:3000/data/data-eunjinlee.json', {
+      method: 'GET'
+    }).then(response => response.json())
+      .then(response => {
+        this.setState({
+          commentList: response.data
+        })
+      })
+  }
+
+  viewComments = () => {
+    this.setState({
+      openComments: !this.state.openComments
+    })
+  }
+
+  getInputValue = (e) => {
+    this.setState({
+      value: e.target.value.trim(),
+    })
+  }
+
+  addComment = (e) => {
+    e.preventDefault()
+    this.setState({
+      commentList: this.state.commentList.concat([
+        {
+          // id: Math.floor(Math.random() * 9000000) + 1000000,
+          id: this.state.commentList.length + 1,
+          userId: 'workoutbutlazy', 
+          text: this.state.value, 
+          like: false,
+        }
+      ]),
+      value: '',
+    })
+    e.target.reset()
+  }
```

Comment:
```suggestion
  addComment = (e) => {
    e.preventDefault()
    const { commentList } = this.state;
    const newCommentList = [...commentList, 
       {
          id: this.state.commentList.length + 1,
          userId: 'workoutbutlazy', 
          text: this.state.value, 
          like: false,
        }
    ]
    
    this.setState({
      commentList: newCommentList,
      value: '',
    })
    
    e.target.reset()
  }
```
- setState의 값으로 지정하는 값은 변수에 담아서 처리해주는 것이 좋습니다.
- 비구조화할당을 생활화 합시다!
- `...` spread 연산자도 학습해보시기 바랍니다.